src/invoices/InvoiceDetails/CancellationModal/CancellationModal.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useCallback } from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  Button,
+  Modal,
+  ModalFooter,
+  TextArea,
+  Row,
+  Col,
+} from '@folio/stripes/components';
+
+const CancellationModal = ({
+  onCancel,
+  onConfirm,
+}) => {
+  const [cancellationNote, setCancellationNote] = useState('');
+
+  const handleChange = useCallback(
+    ({ target: { value } }) => (
+      setCancellationNote(value)
+    ),
+    [],
+  );
+
+  const footer = (
+    <ModalFooter>
+      <Button
+        data-test-confirm-button
```

Comment:
you don't need data-test attrs in case you don't implement bigtest cases - it's deprecated

Reply:
updated